Code/autopkglib/github/__init__.py

```diff
@@ -111,129 +182,26 @@ def call_api(
                 assets)."""
 
         # Compose the URL
```

Comment:
Since this now returns a tuple, please documented what the expected return values of this function are.

Reply:
I am not sure what you mean. 

Do you reffer to `call_api()` method returning a tuple?

It had returned it before my changes and return value is documented in method docstring:

> """Return a tuple of a serialized JSON response and HTTP status code
        from a call to a GitHub API endpoint. Certain APIs return no JSON
        result and so the first item in the tuple (the response) will be None.